src/main/java/io/github/opencubicchunks/cubicchunks/mixin/dasmsets/ForgeSet.java

```diff
@@ -1,21 +1,33 @@
 package io.github.opencubicchunks.cubicchunks.mixin.dasmsets;
 
 import io.github.notstirred.dasm.api.annotations.redirect.redirects.ConstructorToFactoryRedirect;
+import io.github.notstirred.dasm.api.annotations.redirect.redirects.TypeRedirect;
 import io.github.notstirred.dasm.api.annotations.redirect.sets.InterOwnerContainer;
 import io.github.notstirred.dasm.api.annotations.redirect.sets.RedirectSet;
 import io.github.notstirred.dasm.api.annotations.selector.ConstructorMethodSig;
 import io.github.notstirred.dasm.api.annotations.selector.Ref;
 import io.github.opencubicchunks.cubicchunks.movetoforgesourcesetlater.EventConstructorDelegates;
 import io.github.opencubicchunks.cubicchunks.world.level.chunklike.CloAccess;
 import net.minecraft.world.level.chunk.ChunkAccess;
+import net.neoforged.bus.api.Event;
 import net.neoforged.neoforge.event.level.ChunkEvent;
 
 // TODO once redirect sets can be applied conditionally, this should be in the forge sourceset and GlobalSet should no longer extend it
 @RedirectSet
 public interface ForgeSet {
+    @TypeRedirect(from = @Ref(ChunkEvent.Load.class), to = @Ref(Event.class))
```

Comment:
Having a typeredirect AND an interowner for the same source class is technically undefined behaviour I think. I didn't think this would ever be a use-case or have since forgotten, but actually within dasm itself this is perfectly sound.

As interowner only permits to-static redirects, I believe this is always valid as far as the spec is concerned? It probably makes sense to have a more extended discussion about what is allowed and not.

Reply:
> Having a typeredirect AND an interowner for the same source class is technically undefined behaviour I think.

Example 4 in the docs for `ConstructorToFactoryRedirect` already does this